src/index.html

```diff
@@ -1,14 +1,43 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="rectangle">
+      <a href="#" class="mojo-icon">
+        <img src="images/moyo_1.svg"
+        alt="nav-logo"
+        class="nav-logo">
+      </a>
+      <nav class="navigation">
+        <ul class="ul_list">
+          <li class="product_position">
+            <a href="#" class="ul_link is-active">apple
+            </a>
+          </li>
+          <li class="product_position"><a href="#" class="ul_link">samsung</a></li>
+          <li class="product_position"><a href="#" class="ul_link">смартфоны и телефоны</a></li>
+          <li class="product_position">
```

Comment:
если перенес в одном месте, переноси все кейсы
```suggestion
          <li class="product_position">
            <a href="#" class="ul_link">samsung</a>
          </li>
          <li class="product_position"><a href="#" class="ul_link">смартфоны и телефоны</a></li>
          <li class="product_position">
```